---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_arima-2_bases: 259.70 Gbp
data_arima-2_bytes: 139.1 GiB
data_arima-2_coverage: N/A
data_arima-2_links: s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/arima/<br>
data_arima-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/arima/
data_arima-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Glaucomys_sabrinus/mGlaSab2/genomic_data/arima/
data_arima-2_scale: 1.7387
data_arima_bases: 259.70 Gbp
data_arima_bytes: 139.1 GiB
data_arima_coverage: N/A
data_pacbiohifi_bam-2_bases: 163.20 Gbp
data_pacbiohifi_bam-2_bytes: 537.3 GiB
data_pacbiohifi_bam-2_coverage: N/A
data_pacbiohifi_bam-2_links: s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Glaucomys_sabrinus/mGlaSab2/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-2_scale: 0.2829
data_pacbiohifi_bam_bases: 163.20 Gbp
data_pacbiohifi_bam_bytes: 537.3 GiB
data_pacbiohifi_bam_coverage: N/A
data_pacbiohifi_fqgz-2_bases: 159.63 Gbp
data_pacbiohifi_fqgz-2_bytes: 74.8 GiB
data_pacbiohifi_fqgz-2_coverage: N/A
data_pacbiohifi_fqgz-2_links: s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Glaucomys_sabrinus/mGlaSab2/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-2_scale: 1.9882
data_pacbiohifi_fqgz_bases: 159.63 Gbp
data_pacbiohifi_fqgz_bytes: 74.8 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa2date: 2025-10-21
hpa2filesize: 792.6 MiB
hpa2length: '2998687310'
hpa2n50ctg: 86621052
hpa2n50scf: 148354595
hpa2seq: https://s3.amazonaws.com/genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_vgp_HiC_2.0/mGlaSab2.HiC.hap1.20251021.fasta.gz
hpa2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 2 </td><td> 162.00 Mbp </td><td> 2 </td><td> 221.23 Mbp </td></tr><tr><td> 20 </td><td> 4 </td><td> 142.01 Mbp </td><td> 3 </td><td> 181.66 Mbp </td></tr><tr><td> 30 </td><td> 7 </td><td> 119.48 Mbp </td><td> 5 </td><td> 165.46 Mbp </td></tr><tr><td> 40 </td><td> 9 </td><td> 108.56 Mbp </td><td> 7 </td><td> 155.30 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 12 </td><td style="background-color:#88ff88;"> 86.62 Mbp </td><td> 9 </td><td style="background-color:#88ff88;"> 148.35 Mbp </td></tr><tr><td> 60 </td><td> 16 </td><td> 83.46 Mbp </td><td> 11 </td><td> 142.05 Mbp </td></tr><tr><td> 70 </td><td> 20 </td><td> 54.72 Mbp </td><td> 13 </td><td> 111.24 Mbp </td></tr><tr><td> 80 </td><td> 27 </td><td> 33.11 Mbp </td><td> 16 </td><td> 83.63 Mbp </td></tr><tr><td> 90 </td><td> 41 </td><td> 11.48 Mbp </td><td> 21 </td><td> 50.70 Mbp </td></tr><tr><td> 100 </td><td> 356 </td><td> 10.26 Kbp </td><td> 262 </td><td> 10.26 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 356 </th><th> 3.00 Gbp </th><th> 262 </th><th> 3.00 Gbp </th></tr>
  </tfoot>
  </table>
hpa2version: assembly_vgp_HiC_2.0
hpb2date: 2025-10-21
hpb2filesize: 749.0 MiB
hpb2length: '2824073553'
hpb2n50ctg: 85191408
hpb2n50scf: 142777069
hpb2seq: https://s3.amazonaws.com/genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_vgp_HiC_2.0/mGlaSab2.HiC.hap2.20251021.fasta.gz
hpb2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 2 </td><td> 160.31 Mbp </td><td> 2 </td><td> 216.96 Mbp </td></tr><tr><td> 20 </td><td> 4 </td><td> 148.55 Mbp </td><td> 3 </td><td> 174.57 Mbp </td></tr><tr><td> 30 </td><td> 6 </td><td> 109.57 Mbp </td><td> 5 </td><td> 160.31 Mbp </td></tr><tr><td> 40 </td><td> 9 </td><td> 96.22 Mbp </td><td> 6 </td><td> 158.72 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 12 </td><td style="background-color:#88ff88;"> 85.19 Mbp </td><td> 8 </td><td style="background-color:#88ff88;"> 142.78 Mbp </td></tr><tr><td> 60 </td><td> 15 </td><td> 84.20 Mbp </td><td> 10 </td><td> 139.47 Mbp </td></tr><tr><td> 70 </td><td> 19 </td><td> 58.08 Mbp </td><td> 13 </td><td> 106.84 Mbp </td></tr><tr><td> 80 </td><td> 26 </td><td> 35.66 Mbp </td><td> 16 </td><td> 84.41 Mbp </td></tr><tr><td> 90 </td><td> 39 </td><td> 11.82 Mbp </td><td> 19 </td><td> 61.53 Mbp </td></tr><tr><td> 100 </td><td> 281 </td><td> 11.63 Kbp </td><td> 188 </td><td> 11.63 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 281 </th><th> 2.82 Gbp </th><th> 188 </th><th> 2.82 Gbp </th></tr>
  </tfoot>
  </table>
hpb2version: assembly_vgp_HiC_2.0
last_raw_data: 1752775465
last_updated: 1761058619
mds:
- data: species:&nbsp;Glaucomys&nbsp;sabrinus<br>specimem:&nbsp;mGlaSab2<br>projects:<br>&nbsp;&nbsp;-&nbsp;vgp<br>assembled_by_group:&nbsp;Rockefeller<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>combine_for_curation:&nbsp;true<br>hap1:&nbsp;s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_vgp_HiC_2.0/mGlaSab2.HiC.hap1.20251021.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_vgp_HiC_2.0/mGlaSab2.HiC.hap2.20251021.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/s2/mGlaSab2_hap1__s2_heatmap.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/s2/mGlaSab2_hap2__s2_heatmap.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_vgp_HiC_2.0/evaluation/merqury/mGlaSab2_png/<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Glaucomys_sabrinus/mGlaSab2/genomic_data/arima/<br>pipeline:<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.25.0+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy3)<br>&nbsp;&nbsp;-&nbsp;VGP0&nbsp;mitoHiFi&nbsp;workflow&nbsp;(v0.2.2)<br>&nbsp;&nbsp;-&nbsp;VGP1&nbsp;genomic&nbsp;profiling&nbsp;workflow&nbsp;(v0.4)<br>&nbsp;&nbsp;-&nbsp;VGP4&nbsp;HiFi&nbsp;HiC&nbsp;assembly&nbsp;workflow&nbsp;(v0.3.12)<br>&nbsp;&nbsp;-&nbsp;VGP8&nbsp;HiC&nbsp;scaffolding&nbsp;workflow&nbsp;(v3.1)&nbsp;with&nbsp;default&nbsp;mapq=10<br>&nbsp;&nbsp;-&nbsp;VGP9&nbsp;decontamination&nbsp;workflow&nbsp;(v1.1)&nbsp;<br>&nbsp;&nbsp;-&nbsp;Pre-curation&nbsp;worklfow&nbsp;(v1.0beta6)&nbsp;with&nbsp;mapq=10<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;Hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;mGlaSab2&nbsp;resulting&nbsp;in&nbsp;two&nbsp;haplotype&nbsp;assemblies.&nbsp;HiC&nbsp;data&nbsp;were&nbsp;generated&nbsp;using&nbsp;the&nbsp;Arima&nbsp;library&nbsp;prep&nbsp;kit,&nbsp;and&nbsp;require&nbsp;trimming&nbsp;the&nbsp;first&nbsp;5&nbsp;bp&nbsp;off&nbsp;from&nbsp;the&nbsp;5'&nbsp;end&nbsp;of&nbsp;both&nbsp;read&nbsp;files.&nbsp;The&nbsp;restriction&nbsp;enzymes&nbsp;used&nbsp;were&nbsp;the&nbsp;Arima&nbsp;v2&nbsp;kit,&nbsp;so&nbsp;the&nbsp;cutting&nbsp;sites&nbsp;are&nbsp;GATC,&nbsp;GANTC,&nbsp;CTNAG,&nbsp;TTAA.<br>
  ident: md4
  title: assembly_vgp_HiC_2.0/mGlaSab2.yml
mito2date: 2025-10-21
mito2filesize: 5.3 KiB
mito2length: '16545'
mito2n50ctg: 16545
mito2n50scf: 16545
mito2seq: https://s3.amazonaws.com/genomeark/species/Glaucomys_sabrinus/mGlaSab2/assembly_MT_rockefeller/mGlaSab2.MT.20251021.fasta.gz
mito2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 16.55 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 16.55 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 16.55 Kbp </td><td> 1 </td><td> 16.55 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 16.55 Kbp </th><th> 1 </th><th> 16.55 Kbp </th></tr>
  </tfoot>
  </table>
mito2version: assembly_MT_rockefeller
name: Glaucomys sabrinus
name_: Glaucomys_sabrinus
project: ~
short_name: mGlaSab
taxon_id: ''
---
