---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_arima-1_bases: 73.14 Gbp
data_arima-1_bytes: 40.0 GiB
data_arima-1_coverage: N/A
data_arima-1_links: s3://genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Nematolebias_whitei/fNemWhi1/genomic_data/arima/
data_arima-1_scale: 1.7036
data_arima_bases: 73.14 Gbp
data_arima_bytes: 40.0 GiB
data_arima_coverage: N/A
data_pacbiohifi_bam-1_bases: 120.20 Gbp
data_pacbiohifi_bam-1_bytes: 391.1 GiB
data_pacbiohifi_bam-1_coverage: N/A
data_pacbiohifi_bam-1_links: s3://genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Nematolebias_whitei/fNemWhi1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_scale: 0.2862
data_pacbiohifi_bam_bases: 120.20 Gbp
data_pacbiohifi_bam_bytes: 391.1 GiB
data_pacbiohifi_bam_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 118.57 Gbp
data_pacbiohifi_fqgz-1_bytes: 52.7 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Nematolebias_whitei/fNemWhi1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 2.0951
data_pacbiohifi_fqgz_bases: 118.57 Gbp
data_pacbiohifi_fqgz_bytes: 52.7 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2025-10-16
hpa1filesize: 355.6 MiB
hpa1length: '1414218467'
hpa1n50ctg: 8094838
hpa1n50scf: 59381053
hpa1seq: https://s3.amazonaws.com/genomeark/species/Nematolebias_whitei/fNemWhi1/assembly_vgp_HiC_2.0/fNemWhi1.HiC.hap1.20251016.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 5 </td><td> 25.57 Mbp </td><td> 2 </td><td> 114.37 Mbp </td></tr><tr><td> 20 </td><td> 12 </td><td> 20.12 Mbp </td><td> 3 </td><td> 106.93 Mbp </td></tr><tr><td> 30 </td><td> 20 </td><td> 13.49 Mbp </td><td> 4 </td><td> 94.59 Mbp </td></tr><tr><td> 40 </td><td> 32 </td><td> 10.30 Mbp </td><td> 6 </td><td> 68.78 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 48 </td><td style="background-color:#88ff88;"> 8.09 Mbp </td><td> 9 </td><td style="background-color:#88ff88;"> 59.38 Mbp </td></tr><tr><td> 60 </td><td> 67 </td><td> 6.52 Mbp </td><td> 11 </td><td> 58.00 Mbp </td></tr><tr><td> 70 </td><td> 93 </td><td> 4.70 Mbp </td><td> 14 </td><td> 48.66 Mbp </td></tr><tr><td> 80 </td><td> 131 </td><td> 3.01 Mbp </td><td> 16 </td><td> 48.17 Mbp </td></tr><tr><td> 90 </td><td> 193 </td><td> 1.71 Mbp </td><td> 20 </td><td> 32.46 Mbp </td></tr><tr><td> 100 </td><td> 788 </td><td> 15.47 Kbp </td><td> 450 </td><td> 15.47 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 788 </th><th> 1.41 Gbp </th><th> 450 </th><th> 1.41 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_vgp_HiC_2.0
hpb1date: 2025-10-16
hpb1filesize: 355.8 MiB
hpb1length: '1411228583'
hpb1n50ctg: 9155524
hpb1n50scf: 58676569
hpb1seq: https://s3.amazonaws.com/genomeark/species/Nematolebias_whitei/fNemWhi1/assembly_vgp_HiC_2.0/fNemWhi1.HiC.hap2.20251016.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 5 </td><td> 25.49 Mbp </td><td> 1 </td><td> 165.63 Mbp </td></tr><tr><td> 20 </td><td> 11 </td><td> 20.31 Mbp </td><td> 3 </td><td> 71.32 Mbp </td></tr><tr><td> 30 </td><td> 19 </td><td> 16.16 Mbp </td><td> 5 </td><td> 68.06 Mbp </td></tr><tr><td> 40 </td><td> 30 </td><td> 12.25 Mbp </td><td> 7 </td><td> 61.70 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 43 </td><td style="background-color:#88ff88;"> 9.16 Mbp </td><td> 9 </td><td style="background-color:#88ff88;"> 58.68 Mbp </td></tr><tr><td> 60 </td><td> 62 </td><td> 6.85 Mbp </td><td> 11 </td><td> 57.63 Mbp </td></tr><tr><td> 70 </td><td> 86 </td><td> 4.95 Mbp </td><td> 14 </td><td> 52.58 Mbp </td></tr><tr><td> 80 </td><td> 121 </td><td> 3.30 Mbp </td><td> 17 </td><td> 47.00 Mbp </td></tr><tr><td> 90 </td><td> 177 </td><td> 1.86 Mbp </td><td> 21 </td><td> 30.82 Mbp </td></tr><tr><td> 100 </td><td> 594 </td><td> 16.85 Kbp </td><td> 286 </td><td> 16.85 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 594 </th><th> 1.41 Gbp </th><th> 286 </th><th> 1.41 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_vgp_HiC_2.0
last_raw_data: 1751915442
last_updated: 1761947569
mds:
- data: species:&nbsp;Nematolebias&nbsp;whitei<br>specimem:&nbsp;fNemWhi1<br>projects:<br>&nbsp;&nbsp;-&nbsp;vgp<br>assembled_by_group:&nbsp;Rockefeller<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>combine_for_curation:&nbsp;true<br>hap1:&nbsp;s3://genomeark/species/Nematolebias_whitei/fNemWhi1/assembly_vgp_HiC_2.0/fNemWhi1.HiC.hap1.20251016.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Nematolebias_whitei/fNemWhi1/assembly_vgp_HiC_2.0/fNemWhi1.HiC.hap2.20251016.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Nematolebias_whitei/fNemWhi1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/s2/fNemWhi1_hap1__s2_heatmap.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Nematolebias_whitei/fNemWhi1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/s2/fNemWhi1_hap2__s2_heatmap.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Nematolebias_whitei/fNemWhi1/assembly_vgp_HiC_2.0/evaluation/merqury/fNemWhi1_png/<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Nematolebias_whitei/fNemWhi1/genomic_data/arima/<br>pipeline:<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.25.0+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy2)<br>&nbsp;&nbsp;-&nbsp;VGP1&nbsp;genomic&nbsp;profiling&nbsp;workflow&nbsp;(v0.2.2)<br>&nbsp;&nbsp;-&nbsp;VGP4&nbsp;HiFi&nbsp;HiC&nbsp;assembly&nbsp;workflow&nbsp;(v0.3.12)<br>&nbsp;&nbsp;-&nbsp;VGP8&nbsp;HiC&nbsp;scaffolding&nbsp;workflow&nbsp;(v2.2)&nbsp;with&nbsp;mapq=10<br>&nbsp;&nbsp;-&nbsp;VGP9&nbsp;decontamination&nbsp;workflow&nbsp;(v1.1)&nbsp;<br>&nbsp;&nbsp;-&nbsp;Pre-curation&nbsp;worklfow&nbsp;(v1.0beta6)&nbsp;with&nbsp;mapq=10<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;Hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;fNemWhi1&nbsp;resulting&nbsp;in&nbsp;two&nbsp;haplotype&nbsp;assemblies.&nbsp;HiC&nbsp;data&nbsp;were&nbsp;generated&nbsp;using&nbsp;the&nbsp;Arima&nbsp;library&nbsp;prep&nbsp;kit,&nbsp;and&nbsp;require&nbsp;trimming&nbsp;the&nbsp;first&nbsp;5&nbsp;bp&nbsp;off&nbsp;from&nbsp;the&nbsp;5'&nbsp;end&nbsp;of&nbsp;both&nbsp;read&nbsp;files.&nbsp;The&nbsp;restriction&nbsp;enzymes&nbsp;used&nbsp;were&nbsp;the&nbsp;Arima&nbsp;v2&nbsp;kit,&nbsp;so&nbsp;the&nbsp;cutting&nbsp;sites&nbsp;are&nbsp;GATC,&nbsp;GANTC,&nbsp;CTNAG,&nbsp;TTAA.&nbsp;Mitogenome&nbsp;was&nbsp;not&nbsp;assembled&nbsp;in&nbsp;mitoHiFi&nbsp;as&nbsp;no&nbsp;HiFi&nbsp;reads&nbsp;were&nbsp;mapped&nbsp;to&nbsp;the&nbsp;mito&nbsp;reference.&nbsp;There&nbsp;are&nbsp;some&nbsp;2-copy&nbsp;k-mers&nbsp;in&nbsp;the&nbsp;diploid&nbsp;coverage&nbsp;range&nbsp;in&nbsp;hap2.&nbsp;I&nbsp;injected&nbsp;a&nbsp;track&nbsp;in&nbsp;pretextmap&nbsp;to&nbsp;show&nbsp;the&nbsp;location&nbsp;of&nbsp;the&nbsp;potential&nbsp;haplotigs/overlaps&nbsp;identified&nbsp;by&nbsp;purge-dups&nbsp;tool.&nbsp;Curation&nbsp;team&nbsp;could&nbsp;use&nbsp;the&nbsp;track&nbsp;to&nbsp;assess&nbsp;whether&nbsp;they&nbsp;are&nbsp;real&nbsp;haplotigs&nbsp;(note&nbsp;that&nbsp;purge_dups&nbsp;tool&nbsp;tends&nbsp;to&nbsp;over-purge).<br>
  ident: md2
  title: assembly_vgp_HiC_2.0/fNemWhi1.yml
name: Nematolebias whitei
name_: Nematolebias_whitei
project: ~
short_name: fNemWhi
taxon_id: ''
---
