---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_arima-1_bases: 238.85 Gbp
data_arima-1_bytes: 123.9 GiB
data_arima-1_coverage: N/A
data_arima-1_links: s3://genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Perognathus_parvus/mPerPar1/genomic_data/arima/
data_arima-1_scale: 1.7953
data_arima_bases: 238.85 Gbp
data_arima_bytes: 123.9 GiB
data_arima_coverage: N/A
data_pacbiohifi_bam-1_bases: 198.87 Gbp
data_pacbiohifi_bam-1_bytes: 653.5 GiB
data_pacbiohifi_bam-1_coverage: N/A
data_pacbiohifi_bam-1_links: s3://genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Perognathus_parvus/mPerPar1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_scale: 0.2834
data_pacbiohifi_bam_bases: 198.87 Gbp
data_pacbiohifi_bam_bytes: 653.5 GiB
data_pacbiohifi_bam_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 195.19 Gbp
data_pacbiohifi_fqgz-1_bytes: 93.2 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Perognathus_parvus/mPerPar1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 1.9496
data_pacbiohifi_fqgz_bases: 195.19 Gbp
data_pacbiohifi_fqgz_bytes: 93.2 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2025-10-10
hpa1filesize: 560.7 MiB
hpa1length: '2157774171'
hpa1n50ctg: 37691053
hpa1n50scf: 80793895
hpa1seq: https://s3.amazonaws.com/genomeark/species/Perognathus_parvus/mPerPar1/assembly_vgp_HiC_2.0/mPerPar1.HiC.hap1.20251010.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 3 </td><td> 85.98 Mbp </td><td> 2 </td><td> 151.60 Mbp </td></tr><tr><td> 20 </td><td> 6 </td><td> 59.82 Mbp </td><td> 3 </td><td> 137.71 Mbp </td></tr><tr><td> 30 </td><td> 10 </td><td> 47.54 Mbp </td><td> 5 </td><td> 117.87 Mbp </td></tr><tr><td> 40 </td><td> 14 </td><td> 41.98 Mbp </td><td> 7 </td><td> 99.81 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 20 </td><td style="background-color:#88ff88;"> 37.69 Mbp </td><td> 10 </td><td style="background-color:#88ff88;"> 80.79 Mbp </td></tr><tr><td> 60 </td><td> 26 </td><td> 27.30 Mbp </td><td> 12 </td><td> 73.52 Mbp </td></tr><tr><td> 70 </td><td> 35 </td><td> 21.20 Mbp </td><td> 16 </td><td> 62.42 Mbp </td></tr><tr><td> 80 </td><td> 49 </td><td> 11.57 Mbp </td><td> 19 </td><td> 56.91 Mbp </td></tr><tr><td> 90 </td><td> 78 </td><td> 5.04 Mbp </td><td> 25 </td><td> 22.45 Mbp </td></tr><tr><td> 100 </td><td> 447 </td><td> 10.04 Kbp </td><td> 321 </td><td> 10.04 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 447 </th><th> 2.16 Gbp </th><th> 321 </th><th> 2.16 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_vgp_HiC_2.0
hpb1date: 2025-10-10
hpb1filesize: 563.6 MiB
hpb1length: '2200464835'
hpb1n50ctg: 32280039
hpb1n50scf: 79750883
hpb1seq: https://s3.amazonaws.com/genomeark/species/Perognathus_parvus/mPerPar1/assembly_vgp_HiC_2.0/mPerPar1.HiC.hap2.20251010.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 3 </td><td> 77.35 Mbp </td><td> 2 </td><td> 155.25 Mbp </td></tr><tr><td> 20 </td><td> 6 </td><td> 65.37 Mbp </td><td> 3 </td><td> 141.37 Mbp </td></tr><tr><td> 30 </td><td> 10 </td><td> 50.57 Mbp </td><td> 5 </td><td> 125.60 Mbp </td></tr><tr><td> 40 </td><td> 15 </td><td> 44.05 Mbp </td><td> 7 </td><td> 99.91 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 21 </td><td style="background-color:#88ff88;"> 32.28 Mbp </td><td> 10 </td><td style="background-color:#88ff88;"> 79.75 Mbp </td></tr><tr><td> 60 </td><td> 29 </td><td> 25.11 Mbp </td><td> 12 </td><td> 70.72 Mbp </td></tr><tr><td> 70 </td><td> 38 </td><td> 20.71 Mbp </td><td> 16 </td><td> 61.84 Mbp </td></tr><tr><td> 80 </td><td> 52 </td><td> 11.90 Mbp </td><td> 19 </td><td> 58.39 Mbp </td></tr><tr><td> 90 </td><td> 87 </td><td> 3.84 Mbp </td><td> 24 </td><td> 38.53 Mbp </td></tr><tr><td> 100 </td><td> 387 </td><td> 17.34 Kbp </td><td> 249 </td><td> 17.34 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 387 </th><th> 2.20 Gbp </th><th> 249 </th><th> 2.20 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_vgp_HiC_2.0
last_raw_data: 1756838537
last_updated: 1760125245
mds:
- data: species:&nbsp;Perognathus&nbsp;parvus<br>specimem:&nbsp;mPerPar1<br>projects:<br>&nbsp;&nbsp;-&nbsp;vgp<br>assembled_by_group:&nbsp;Rockefeller<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>combine_for_curation:&nbsp;true<br>hap1:&nbsp;s3://genomeark/species/Perognathus_parvus/mPerPar1/assembly_vgp_HiC_2.0/mPerPar1.HiC.hap1.20251010.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Perognathus_parvus/mPerPar1/assembly_vgp_HiC_2.0/mPerPar1.HiC.hap2.20251010.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Perognathus_parvus/mPerPar1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/s2/mPerPar1_hap1__s2_heatmap.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Perognathus_parvus/mPerPar1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/s2/mPerPar1_hap2__s2_heatmap.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Perognathus_parvus/mPerPar1/assembly_vgp_HiC_2.0/evaluation/merqury/mPerPar1_png/<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Perognathus_parvus/mPerPar1/genomic_data/arima/<br>pipeline:<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.25.0+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy2)<br>&nbsp;&nbsp;-&nbsp;VGP0&nbsp;mitoHiFi&nbsp;workflow&nbsp;(v0.2.2)<br>&nbsp;&nbsp;-&nbsp;VGP1&nbsp;genomic&nbsp;profiling&nbsp;workflow&nbsp;(v0.4)<br>&nbsp;&nbsp;-&nbsp;VGP4&nbsp;HiFi&nbsp;HiC&nbsp;assembly&nbsp;workflow&nbsp;(v0.3.12)<br>&nbsp;&nbsp;-&nbsp;VGP8&nbsp;HiC&nbsp;scaffolding&nbsp;workflow&nbsp;(v2.2)&nbsp;with&nbsp;mapq=10<br>&nbsp;&nbsp;-&nbsp;VGP9&nbsp;decontamination&nbsp;workflow&nbsp;unpublished&nbsp;using&nbsp;FCS&nbsp;(0.5.5+galaxy0)&nbsp;<br>&nbsp;&nbsp;-&nbsp;Pre-curation&nbsp;worklfow&nbsp;(v1.0beta6)&nbsp;with&nbsp;mapq=10<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;Hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;mPerPar1&nbsp;resulting&nbsp;in&nbsp;two&nbsp;haplotype&nbsp;assemblies.&nbsp;HiC&nbsp;data&nbsp;were&nbsp;generated&nbsp;using&nbsp;the&nbsp;Arima&nbsp;library&nbsp;prep&nbsp;kit,&nbsp;and&nbsp;require&nbsp;trimming&nbsp;the&nbsp;first&nbsp;5&nbsp;bp&nbsp;off&nbsp;from&nbsp;the&nbsp;5'&nbsp;end&nbsp;of&nbsp;both&nbsp;read&nbsp;files.&nbsp;The&nbsp;restriction&nbsp;enzymes&nbsp;used&nbsp;were&nbsp;the&nbsp;Arima&nbsp;v2&nbsp;kit,&nbsp;so&nbsp;the&nbsp;cutting&nbsp;sites&nbsp;are&nbsp;GATC,&nbsp;GANTC,&nbsp;CTNAG,&nbsp;TTAA.<br>
  ident: md4
  title: assembly_vgp_HiC_2.0/mPerPar1.yml
mito1date: 2025-10-10
mito1filesize: 5.2 KiB
mito1length: '16326'
mito1n50ctg: 16326
mito1n50scf: 16326
mito1seq: https://s3.amazonaws.com/genomeark/species/Perognathus_parvus/mPerPar1/assembly_MT_rockefeller/mPerPar1.MT.20251010.fasta.gz
mito1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 16.33 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 16.33 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 16.33 Kbp </td><td> 1 </td><td> 16.33 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 16.33 Kbp </th><th> 1 </th><th> 16.33 Kbp </th></tr>
  </tfoot>
  </table>
mito1version: assembly_MT_rockefeller
name: Perognathus parvus
name_: Perognathus_parvus
project: ~
short_name: mPerPar
taxon_id: ''
---
