---
alt1date: 2024-05-02
alt1filesize: 2.5 GiB
alt1length: '10645492134'
alt1n50ctg: 0
alt1n50scf: 0
alt1seq: https://s3.amazonaws.com/genomeark/species/Triturus_cristatus/aTriCri1/assembly_cambridge/aTriCri1.alt.asm.20240502.fasta.gz
alt1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 2039 </td><td> 433.05 Kbp </td><td> 2039 </td><td> 433.05 Kbp </td></tr><tr><td> 20 </td><td> 23870 </td><td> 69.52 Kbp </td><td> 23870 </td><td> 69.52 Kbp </td></tr><tr><td> 30 </td><td> 69361 </td><td> 43.85 Kbp </td><td> 69361 </td><td> 43.85 Kbp </td></tr><tr><td> 40 </td><td> 135821 </td><td> 28.99 Kbp </td><td> 135821 </td><td> 28.99 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 60 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 70 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 80 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 90 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 100 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr></tbody>
  <tfoot>
  <tr><th> 0.438x </th><th> 179407 </th><th> 10.65 Gbp </th><th> 179407 </th><th> 10.65 Gbp </th></tr>
  </tfoot>
  </table>
alt1version: assembly_cambridge
assembly_status: <em style="color:orangered">Draft</em>
common_name: warty newt
data_arima-1_bases: 918.45 Gbp
data_arima-1_bytes: 278.9 GiB
data_arima-1_coverage: 37.75x
data_arima-1_links: s3://genomeark/species/Triturus_cristatus/aTriCri1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Triturus_cristatus/aTriCri1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Triturus_cristatus/aTriCri1/genomic_data/arima/
data_arima-1_scale: 3.0664
data_arima_bases: 918.45 Gbp
data_arima_bytes: 278.9 GiB
data_arima_coverage: 37.75x
data_pacbiohifi_bam-1_bases: 768.41 Gbp
data_pacbiohifi_bam-1_bytes: 291.6 GiB
data_pacbiohifi_bam-1_coverage: 31.59x
data_pacbiohifi_bam-1_links: s3://genomeark/species/Triturus_cristatus/aTriCri1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Triturus_cristatus/aTriCri1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Triturus_cristatus/aTriCri1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_scale: 2.4542
data_pacbiohifi_bam_bases: 768.41 Gbp
data_pacbiohifi_bam_bytes: 291.6 GiB
data_pacbiohifi_bam_coverage: 31.59x
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genbank_alt: aTriCri1:GCA_964204665.1
genbank_pri: aTriCri1:GCA_964204655.1
genome_size: 24327750000
genome_size_display: 24.33 Gbp
genome_size_method: GoaT
last_raw_data: 1714045361
last_updated: 1715416361
metadata: species:<br>&nbsp;&nbsp;short_name:&nbsp;aTriCri<br>&nbsp;&nbsp;name:&nbsp;Triturus&nbsp;cristatus<br>&nbsp;&nbsp;taxon_id:&nbsp;8323<br>&nbsp;&nbsp;common_name:&nbsp;warty&nbsp;newt<br>&nbsp;&nbsp;order:<br>&nbsp;&nbsp;&nbsp;&nbsp;name:&nbsp;Caudata<br>&nbsp;&nbsp;family:<br>&nbsp;&nbsp;&nbsp;&nbsp;name:&nbsp;Salamandridae<br>&nbsp;&nbsp;individuals:<br>&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;short_name:&nbsp;aTriCri1<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;biosample_id:&nbsp;SAMEA112468129<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sex:&nbsp;female<br>&nbsp;&nbsp;genome_size:&nbsp;24327750000<br>&nbsp;&nbsp;genome_size_method:&nbsp;GoaT<br>&nbsp;&nbsp;project:&nbsp;[&nbsp;vgp&nbsp;]<br>
mito1date: 2024-05-02
mito1filesize: 5.3 KiB
mito1length: '16543'
mito1n50ctg: 0
mito1n50scf: 0
mito1seq: https://s3.amazonaws.com/genomeark/species/Triturus_cristatus/aTriCri1/assembly_MT_cambridge/aTriCri1.MT.20240502.fasta.gz
mito1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 20 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 30 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 40 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 0 </td><td style="background-color:#ff8888;">  </td><td> 0 </td><td style="background-color:#ff8888;">  </td></tr><tr><td> 60 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 70 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 80 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 90 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr><tr><td> 100 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr></tbody>
  <tfoot>
  <tr><th> 0.000x </th><th> 1 </th><th> 16.54 Kbp </th><th> 1 </th><th> 16.54 Kbp </th></tr>
  </tfoot>
  </table>
mito1version: assembly_MT_cambridge
name: Triturus cristatus
name_: Triturus_cristatus
pri1date: 2024-05-02
pri1filesize: 6.2 GiB
pri1length: '23625839738'
pri1n50ctg: 7946480
pri1n50scf: 1857603958
pri1seq: https://s3.amazonaws.com/genomeark/species/Triturus_cristatus/aTriCri1/assembly_cambridge/aTriCri1.pri.asm.20240502.fasta.gz
pri1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 96 </td><td> 20.36 Mbp </td><td> 1 </td><td> 2.47 Gbp </td></tr><tr><td> 20 </td><td> 238 </td><td> 14.96 Mbp </td><td> 2 </td><td> 2.41 Gbp </td></tr><tr><td> 30 </td><td> 422 </td><td> 11.88 Mbp </td><td> 4 </td><td> 2.15 Gbp </td></tr><tr><td> 40 </td><td> 646 </td><td> 9.89 Mbp </td><td> 5 </td><td> 1.92 Gbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 921 </td><td style="background-color:#88ff88;"> 7.95 Mbp </td><td> 6 </td><td style="background-color:#88ff88;"> 1.86 Gbp </td></tr><tr><td> 60 </td><td> 1263 </td><td> 6.38 Mbp </td><td> 7 </td><td> 1.68 Gbp </td></tr><tr><td> 70 </td><td> 1696 </td><td> 4.96 Mbp </td><td> 9 </td><td> 1.26 Gbp </td></tr><tr><td> 80 </td><td> 2273 </td><td> 3.53 Mbp </td><td> 11 </td><td> 1.14 Gbp </td></tr><tr><td> 90 </td><td> 3167 </td><td> 1.97 Mbp </td><td> 18 </td><td> 100.70 Mbp </td></tr><tr><td> 100 </td><td> 0 </td><td>  </td><td> 0 </td><td>  </td></tr></tbody>
  <tfoot>
  <tr><th> 0.971x </th><th> 6447 </th><th> 23.62 Gbp </th><th> 1532 </th><th> 23.63 Gbp </th></tr>
  </tfoot>
  </table>
pri1version: assembly_cambridge
project:
- vgp
short_name: aTriCri
taxon_id: 8323
---
